### PR TITLE
Add MoreCulling Bug Fixes

### PR DESCRIPTION
**Pull request by Imzxhir**

### CODE REVIEW

File: en-us/changed-options.md [hunk 1]
```diff
@@ -67,6 +67,11 @@ Fabulously Optimized includes some mods that fix vanilla bugs.
 | [MC-235035](https://bugs.mojang.com/browse/MC-235035) | Sleeping in a custom dimension with "natural" set to false causes crash    | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)             |
 | [MC-237493](https://bugs.mojang.com/browse/MC-237493) | Telemetry cannot be disabled                                               | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)   |
 | [MC-249059](https://bugs.mojang.com/browse/MC-249059) | Loading terrain screen cannot close before 2 seconds have passed           | [Debugify](https://curseforge.com/minecraft/mc-mods/debugify)             |
+| [MC-249693](https://bugs.mojang.com/browse/MC-249693) | Chorus plant blocks explicitly define cullface arguments for faces which end up being obscured by other chorus plants. |  [MoreCulling](https://www.curseforge.com/minecraft/mc-mods/moreculling)  |
```
Comment (Madis0) [inline en-us/changed-options.md:70]: Other titles don't have periods in the end, these shouldn't have either.

File: en-us/changed-options.md [hunk 2]
```diff
@@ -67,6 +67,11 @@ Fabulously Optimized includes some mods that fix vanilla bugs.
...
+| [MC-238739](https://bugs.mojang.com/browse/MC-238739) | Bamboo has cullface specified for the top and bottom faces, like many other blocks do.  | [MoreCulling](https://www.curseforge.com/minecraft/mc-mods/moreculling) |
```
Comment (Madis0) [inline en-us/changed-options.md:71]: Please sort all bugs by bug number.

**Review by Madis0 — CHANGES_REQUESTED** [review]

File: en-us/changed-options.md (hunk 1 above)
Comment (RaptaG) [reply en-us/changed-options.md:74]: He means: Remove the dots

Comment (RaptaG) [reply en-us/changed-options.md:74]: I made a PR in your fork fixing that

**Comment by Madis0** [comment]
Thanks!

**Pull request merged**